packages/theme-default/src/components/Sidebar/SidebarGroup.tsx

```diff
@@ -115,10 +115,8 @@ export function SidebarGroup(props: SidebarItemProps) {
         onClick={e => {
           if (item.link) {
             navigate(withBase(normalizeHref(item.link)));
-            collapsed && toggleCollapse(e);
-          } else {
-            collapsible && toggleCollapse(e);
           }
+          collapsible && toggleCollapse(e);
```

Comment:
This looks the same as the previous code. I think it is refactoring not fixing the issue